module-1/lab-mysql-select/your-code/create.sql

```diff
@@ -0,0 +1,46 @@
+CHALLENGE 1
+
```

Comment:
En esta primer tarea el enunciado dice: `write a MySQL SELECT query that joins various tables`, el select esta bien hecho solo falta poner los joins con las tablas, así:
```
select a.au_id as "AUTHOR ID", au_lname as "LAST NAME", au_fname as "FIRST NAME", t.title as TITLE, p.pub_name as PUBLISHER
from authors a
inner join titleauthor ta on a.au_id=ta.au_id
inner join titles t on ta.title_id=t.title_id
inner join publishers p on t.pub_id=p.pub_id;
```